### PR TITLE
Add 2FA announcement post

### PR DESCRIPTION
**Pull request by chasballew**
@krallin @fancyremarker

### CODE REVIEW

File: content/posts/mfa.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: "2-Factor Authentication!"
+excerpt: "Easily secure your Aptible accounts."
+author_name: Thomas Orozco
+author_email: thomas@aptible.com
+author_id: thomas
+posted: 2016-05-19
+section: Blog
+posts: true
+---
+We're happy to announce that two-factor authentication (2FA) is available for all users and account types in the Aptible dashboard and CLI! Multifactor authentication is a best practice that adds an additional layer of security on top of the normal username and password you use to verify your identity. You can enable it in your Aptible user settings.
```
Comment (krallin) [inline content/posts/mfa.md]: @chasballew perhaps we could include the link here (https://dashboard.aptible.com/settings/protected/admin)?

File: content/posts/mfa.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## How does it work?
+
+Aptible 2-factor authentication implements the Time-based One-time Password (TOTP) algorithm specified in RFC 6238. We currently support the virtual token form factor - [Google Authenticator][0] is an excellent, free app. We do not currently support SMS or hardware tokens.
```
Comment (krallin) [inline content/posts/mfa.md]: The sentence on Google Authenticator read a little weird to me (as in it felt like it was missing a word or two at the end). Might have to do with not being a native speaker, but I would perhaps have used e.g.:

> Google Authenticator is an excellent free app that implements it.

File: content/posts/mfa.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+When enabled, 2FA protects access to your Aptible account via the dashboard, CLI, and API. 2FA does not restrict Git pushes - these are still authenticated by your SSH key. In some cases, you may not push code with your own user credentials. For example if you deploy with a CI service such as Travis or Circle, you may have only a robot user performing deploys. If so, we encourage you to remove SSH keys from your Aptible user account.
```
Comment (krallin) [inline content/posts/mfa.md]: I think "you may" in "you may have only a robot user..." makes the sentence a little complicated, and might be superfluous considering we're discussing a hypothetical example in the first place. 

Perhaps we could edit:

> For example if you deploy with a CI service such as Travis or Circle, you may have only a robot user performing deploys

To:

> For example if you deploy with a CI service such as Travis or Circle and perform all deploys via a robot user.

**Comment by krallin** [comment]
Thanks a lot for fleshing out the blog post @chasballew! I left a couple nits here and there. Feel free to either incorporate or discard :)

**Comment by chasballew** [comment]
Rebasing, merging, and deploying

**Pull request merged**